src/axom/spin/internal/linear_bvh/build_radix_tree.hpp

```diff
@@ -464,13 +448,22 @@ void propagate_aabbs(RadixTree<FloatType, NDIMS>& data, int allocatorID)
 
   int32* counters_ptr = axom::allocate<int32>(inner_size, allocatorID);
 
+  using PointType = primal::Point<FloatType, NDIMS>;
+
+  PointType* min_range = axom::allocate<PointType>(inner_size, allocatorID);
+  PointType* max_range = axom::allocate<PointType>(inner_size, allocatorID);
```

Comment:
It's more that we can't access BoundingBox's internal floats/doubles in a mutable fashion, so we can't use atomic operations on them. I guess to avoid the extra allocations, we could also reinterpret_cast `m_inner_aabbs` to an array of `primal::Point<>`s and access them that way.